Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Trixi"
 uuid = "a7f1ee26-1774-49b1-8366-f1abc58fbfcb"
 authors = ["Michael Schlottke-Lakemper <mschlott@math.uni-koeln.de>", "Gregor Gassner <ggassner@uni-koeln.de>", "Hendrik Ranocha <mail@ranocha.de>", "Andrew R. Winters <andrew.ross.winters@liu.se>"]
-version = "0.3.22"
+version = "0.3.23"
```

Comment:
Are we sure a Julia version bump does not require a minor version bump? I am thinking about the following situation:

A user has a custom Julia package `Foo`. The package `Foo` has Trixi v0.3 as a dependency. `Foo` is then installed on Julia 1.5 or updated on Julia 1.5 from a previous version. Will `Pkg` be smart enough to figure out that it is only allowed to install patch releases up to `.22`?

Reply:
Yes, `Pkg` is smart enough for that.